cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1115,28 +1131,74 @@ function Verify-WorkerServices {
   Log_Todo "run more verification commands."
 }
 
+function DownloadAndInstall-Crictl {
+  $CRICTL_VERSION = "v1.16.1"
+  $CRICTL_SHA256 = "7d092dcb3b1af2edf75477d5d049a70e8c0d1ac8242b1dff2de7e6aa084e3615"
+
+  # Assume that presence of crictl.exe indicates that the crictl binaries
+  # were already previously downloaded to this node.
+  if (-not (ShouldWrite-File ${env:NODE_DIR}\crictl.exe)) {
+    return
+  }
+  $tmp_dir = 'C:\crictl_tmp'
+  New-Item $tmp_dir -ItemType 'directory' -Force | Out-Null
+
+  $url = ('https://github.com/kubernetes-sigs/cri-tools/releases/' +
```

Comment:
could you put it in a gs bucket? similarly to other binaries, like stackdriver, or cni

Reply:
CNI is not in a gcs bucket.

Reply:
if it is not now, it needs to be released in gcs soon.

Reply:
Why?

Reply:
GitHub does have download quotas; using GCS ensures usage is billed instead of blocked

Reply:
production users should probably mirror things they will pull very frequently sure, but github is 100% reasonable for upstream binary releases.